command/intention/create/create.go

```diff
@@ -226,3 +250,7 @@ Usage: consul intention create [options] -file FILE...
 
   Additional flags and more advanced use cases are detailed below.
 `
+// todo: figure out where these normally go
+const consulSrcType = "consul"
+const extURISrcType = "ext-uri"
+const extTrustDomainSrcType = "ext-trust-domain"
```

Comment:
I'd just import the ones from structs package

Reply:
Oh wait these are abbreviated. Is there a reason for not using the same thing? Is seems an odd UX that we'd show a different string in the API when displaying the intentions than the one you have to specify on the CLI?

Reply:
I was using abbreviated to make it shorter. Agree on making them consist though. Did you have an opinion on `ext` vs. `external`?

Reply:
In general HashiCorp tools tend to prefer explicit over abbreviated. I don't mind a great deal but `external` feels more consistent to me.